scripts/background.js

```diff
@@ -1,13 +1,22 @@
 chrome.tabs.onUpdated.addListener((tabId, changeInfo, tab) => {
   const tabUrl = tab.url ?? tab.pendingUrl;
   if (
-    changeInfo.status === 'complete' &&
+    changeInfo.status === "complete" &&
     tabUrl &&
-    (tabUrl.includes('linkedin.com/feed') || tabUrl.includes('linkedin.com/in'))
+    tabUrl.includes("linkedin.com/jobs/search")
```

Comment:
Updated this - we only want to remove elements from the DOm that match specific classes on the job search pages, otherwise there's a risk of accidentally removing unrelated elements.

Reply:
On the `/search` route, I'm injecting the `search.css` file, which I renamed to match the route.